src/Tickets/Commerce/Abstract_Order.php

```diff
@@ -133,4 +135,17 @@ public function get_purchaser_data( $data ) {
 		 */
 		return apply_filters( 'tec_tickets_commerce_order_purchaser_data', $purchaser, $data );
 	}
+
+	/**
+	 * Get the order Gateway Admin URL link by order ID
+	 *
+	 * @since TBD
+	 *
+	 * @param \WP_Post $order The order post object.
+	 *
+	 * @return ?string
+	 */
+	public function get_gateway_admin_url_by_order( \WP_Post $order ): ?string {
```

Comment:
```suggestion
	public function get_gateway_dashboard_url_by_order( \WP_Post $order ): ?string {
```

Reply:
I feel `dashboard_url` makes more sense for 3rd party sites instead of `admin_url` which we use a lot within WP.